src/ocl/dropoutocl.cpp

```diff
@@ -135,15 +135,15 @@ void DropoutDescriptor::InitPRNGState(Handle& handle,
     size_t states_num = prng_stateSizeInBytes / sizeof(prngStates);
     size_t wk_grp_num = std::min(size_t(MAX_PRNG_STATE / 256), (states_num + 255) / 256);
 
-    std::string network_config = "initprngs-" + std::to_string(states_num) + "x" +
-                                 std::to_string(sizeof(prngStates)) + "x" +
-                                 std::to_string(rng_mode) + "x" + std::to_string(prng_seed) + "x" +
-                                 std::to_string(wk_grp_num);
+    std::string network_config = "initprngs-" + std::to_string(sizeof(prngStates)) + "x" +
```

Comment:
Please use `sdt::ostringstream` instead of concatenations if there are more than two strings. It's both faster and easier to read.

Reply:
Thank you. I would do this, but the rest of the file (as well as other files such as `softmaxocl.cpp`) use this style of code. Would it be ok to not match the style of the rest of the code?

Reply:
Yes, otherwise it would never really change.